easybuild/easyconfigs/p/pybind11/pybind11-2.4.3-fix-get-include.patch

```diff
@@ -0,0 +1,23 @@
+The current function doesn't take into account that a custom install prefix might have been set
+at installation time which we do in EB.
+Insert a quick hack to make it work as it should (fixed in 2.5.0)
+diff --git a/pybind11/__init__.py b/pybind11/__init__.py
+index c625e8c..5c2e3bd 100644
+--- a/pybind11/__init__.py
++++ b/pybind11/__init__.py
+@@ -27,10 +27,15 @@ def get_include(user=False):
+ 
+         dist_cobj = dist.get_command_obj('install', create=True)
+ 
++        # Very crude hack to get install prefix from <prefix>/lib/python3.7/site-packages/pybind11
++        prefix = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))))
```

Comment:
@Flamefire not sure if it is necessary here, but `__file__` depends on the location from where it has been called, so maybe this is safer?
```suggestion
+        prefix = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))))

```
also, maybe use pathlib instead?
```
from pathlib import Path
prefix = Path(__file__).resolve().parents[4]
```

Reply:
Good suggestion, that looks better. Fixed the commit